core/arch/arm/arm.mk

```diff
@@ -179,6 +181,7 @@ core-platform-cppflags += $(arm64-platform-cppflags)
 core-platform-cflags += $(arm64-platform-cflags)
 core-platform-cflags += $(arm64-platform-cflags-generic)
 core-platform-cflags += $(arm64-platform-cflags-no-hard-float)
+core-platform-cflags += $(arm64-platform-cflags-no-eh-frame)
```

Comment:
This affects the TEE core (`tee.elf`) as well, not only ldelf, therefore you could (and should) remove `.eh_frame` from the /DISCARD/ output section in `core/arch/arm/kernel/kern.ld.S` since it becomes useless.